docs/DafnyRef/Options.txt

```diff
@@ -487,12 +483,9 @@ Usage: Dafny [ option ... ] [ filename ... ]
           target.
       2 (default) - As in 1, but only resolve method bodies in
           non-included Dafny sources.
-  /library:<file>
```

Comment:
Interesting, what was the root cause of having `/library` duplicated?

Reply:
A coding mistake? I don't think I understand the question.

Reply:
I'm just not seeing where the mistake was that the PR seems to fix. Just trying to improve my own understanding. :)

Reply:
It declared the documentation both in the old way and in the new but support legacy way.